test.js

```diff
@@ -25,3 +26,7 @@ test('min and max arguments set graph range', t => {
 	t.is(fn([10, 11, 12, 13], {min: 0}), '▇▇██');
 	t.is(fn([10, 20, 30, 40, 50], {max: 100}), '▁▂▃▄▄');
 });
+
+test('colored graph', t => {
+	t.is(fn([1, 2, 3, 4, 5, 6, 7, 8], {style: 'fire'}), colors.fg.getRgb(5, 5, 3) + '▁' + colors.fg.getRgb(5, 5, 4) + '▂' + colors.fg.getRgb(5, 5, 0) + '▃' + colors.fg.getRgb(5, 4, 0) + '▄' + colors.fg.getRgb(5, 3, 0) + '▅' + colors.fg.getRgb(5, 2, 0) + '▆' + colors.fg.getRgb(5, 1, 0) + '▇' + colors.fg.getRgb(5, 0, 0) + '█');
```

Comment:
Test isn't very clean to read.

```js
test('colored graph', t => {
	const graph = [`${colors.fg.getRgb(5, 5, 3)}▁`, `${colors.fg.getRgb(5, 5, 4)}▃`, `${colors.fg.getRgb(5, 5, 0)}▃`, `${colors.fg.getRgb(5, 4, 0)}▄`, `${colors.fg.getRgb(5, 3, 0)}▅`, `${colors.fg.getRgb(5, 2, 0)}▆`, `${colors.fg.getRgb(5, 1, 0)}▇`, `${colors.fg.getRgb(5, 0, 0)}█`];

	t.is(fn([1, 2, 3, 4, 5, 6, 7, 8], {style: 'fire'}), graph.join(''));
});
```

Not sure wether this is that much better though, but you should definitely use [template strings](https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/template_strings). Let @sindresorhus decide how he wants this :).